omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -56,16 +56,29 @@ export default class ObsAcrossEncountersController {
   }
 
   fetchEncounters() {
-    this.openmrsRest.get("encounter", {
-      patient: this.config.patientUuid,
-      encounterType: this.config.encounterType ? this.config.encounterType : null,
-      v: 'custom:(uuid,encounterDatetime,obs:(id,uuid,value,concept:(id,uuid,name:(display),datatype:(uuid)),groupMembers:(id,uuid,display,value,concept:(id,uuid,name:(display),datatype:(uuid))))',
-      limit: this.getMaxRecords(),
-      fromdate: this.widgetsCommons.maxAgeToDate(this.config.maxAge),
-      order: this.order
-    }).then((response) => {
-      this.parseEncounters(response.results);
-    });
+    let promises = [];
+    let encounterTypes = this.config.encounterType ? this.config.encounterType.replace(' ', '').split(',') : [null];
+    if (encounterTypes !== null && encounterTypes.length > 0) {
```

Comment:
We still need to support the case where encounterType is not specified....  I think we've lost that here?
https://wiki.openmrs.org/display/docs/Patient+Summary+Widget+Documentation#PatientSummaryWidgetDocumentation-conceptencountersObsAcrossEncounters

Reply:
@mogoodrich, this is still supported by the `[null]` array value.

Reply:
Ah, I missed that, thanks @Ruhanga

Reply:
Might be worth adding a comment to that effect since it's easy to miss.

Reply:
@Ruhanga also this will have to be documented on the wiki.